packages/cashier/src/components/recent-transaction/recent-transaction.tsx

```diff
@@ -120,6 +43,110 @@ const RecentTransaction = observer(() => {
             </div>
         </div>
     );
+
+    if (!crypto_transactions.length) {
+        return (
+            <SideNoteContainer>
+                <Text as='p' line_height='s' size='xs'>
+                    <Localize i18n_default_text='No recent transactions.' />
+                </Text>
+            </SideNoteContainer>
+        );
+    }
+
+    const { address_hash, transaction_hash, transaction_type, status_code, submit_date, confirmations } =
+        crypto_transactions[0];
+    const status = getStatus(transaction_hash, transaction_type, status_code);
+    const submit_date_moment = epochToMoment(submit_date).format('MMM D, YYYY');
+    const transaction_type_display_text = transaction_type[0].toUpperCase() + transaction_type.slice(1);
+    const address_hash_display_value = `${address_hash.substring(0, 4)}....${address_hash.substring(
+        address_hash.length - 4
+    )}`;
+
+    const amount = crypto_transactions[0].amount;
+
+    let confirmation_label = '-';
+
+    if (transaction_type === 'deposit') {
+        if (status_code === 'CONFIRMED') {
+            confirmation_label = localize('Confirmed');
+        } else {
+            confirmation_label = confirmations ? `${confirmations}` : localize('Pending');
+        }
+    }
```

Comment:
Should this be moved to `constants` since it's repeated `crypto-transaction-renderer.tsx` as well?

Reply:
@nijil-deriv Will move it to the `getStatus` method 🙇🏻 Although all of these files are going to be removed soon 👀